krabcake/rs_hello/src/lib.rs

```diff
@@ -263,18 +257,12 @@ fn if_addr_tracked_then<T>(addr: vg_addr, process_tag: impl FnOnce(Tag) -> T) ->
     None
 }
 
-fn if_addr_has_stack_then<T>(
-    addr: vg_addr,
-    process_stack: impl FnOnce(&mut Vec<Item>) -> T,
-) -> Option<T> {
+#[no_mangle]
+pub extern "C" fn rs_client_set_context(ctx: Context) {
     unsafe {
-        for entry in &mut STACKS {
-            if entry.0 == addr {
-                return Some(process_stack(&mut entry.1));
-            }
-        }
+        vgPlain_dmsg("lib.rs: normalizing output...\n\0".as_ptr() as *const c_char);
```

Comment:
This is issuing the same message, regardless of what the value of `ctx.normalize_output` is, right?

That is fine for the short term, given the current valgrind cmdline interface above (i.e. you only ever *call* this method solely if that boolean is set to true).

But if we ever add any *other* context parmeters, then this message will need to be adapted accordingly.

Reply:
>you only ever call this method solely if that boolean is set to true

correct